docs/docs/configuration/kafka-and-zookeeper.md

```diff
@@ -0,0 +1,90 @@
+zookeeper.base.sleep.time        | storage.retrySleep             | time to wait between subsequent retries in seconds                         | 1 000
+zookeeper.root                   | storage.pathPrefix             | perfix for Hermes data (if not specified in connection string)             | /hermes
+zookeeper.cache.thread.pool.size | n/a                            | size of thread pool used by objects cache (like topics, subscriptions etc) | 5
+zookeeper.authorization.enabled  | n/a                            | enable Zookeeper authorization                                             | false
+zookeeper.authorization.scheme   | storage.authorization.scheme   | authorization scheme                                                       | digest
+zookeeper.authorization.user     | storage.authorization.user     | username                                                                   | user
+zookeeper.authorization.password | storage.authorization.password | password                                                                   | password
+
+## Kafka
+
+### Single Kafka cluster
+
+In simple case, Hermes is connected to just one Kafka cluster. Frontend and Consumers connect to Kafka to publish
+and pull messages. Management connects to Kafka to manage existing topics and initiate retransmissions.
+
+Frontend and Consumers options:
+
+Option                         | Description                                                                              | Default value
+------------------------------ | ---------------------------------------------------------------------------------------- | --------------
+kafka.broker.list              | list of all brokers in the cluster (or at least some contact points); separated with ',' | localhost:9092
+kafka.namespace                | namespace is a prefix prepended to all Kafka topics and consumer groups used by Hermes   | <empty>
+kafka.zookeeper.connect.string | [Consumers only] connection string to Kafka Zookeeper                                    | localhost:2181
+kafka.cluster.name             | name of Kafka cluster (relevant only when connecting to multiple clusters)               | primary
+
+Zookeeper connection specific options (retries etc) are read from Metadata Zookeeper options.
+
+Management module can connect to multiple Kafka clusters at once (see [section below](#multiple-kafka-clusters)), thus
+when specifying connection option is done per cluster. Simple configuration for single cluster looks following:
+
+```yaml
+kafka:
+  defaultNamespace: // namespace shared by all clusters, default: <empty>
+  clusters:
+    -
+      clusterName: // name of cluster, can be any arbitrary string, default: primary
+      connectionString: // connection string to cluster Zookeeper, default: localhost:2181
+```
+
+### Multiple Kafka clusters
+
+Hermes can be configured to publish and read messages to/from multiple Kafka clusters. This is the scenario we use for
```

Comment:
I would replace the second sentence to something like this: We use this feature on production environment where we have separated kafka clusters in different data centers.